plugins/mobile_channel_header_plug/mobile_channel_header_plug.jsx

```diff
@@ -30,7 +30,7 @@ export default class MobileChannelHeaderPlug extends React.PureComponent {
         return (
             <div
                 className='navbar-toggle navbar-right__icon pull-right'
-                onClick={plug.action}
+                onClick={() => plug.action(this.props.channel, this.props.channelMember)}
```

Comment:
Can we have this in a function?
Also, if possible, please add unit test (jest) so that such function is called on 'click' and called with parameters of channel and channelMember.

Reply:
> Can we have this in a function?

Sure, I will update as soon as I find a minute or two.

> Also, if possible, please add unit test (jest) so that such function is called on 'click' and called with parameters of channel and channelMember.

I fear that I do not have the resources to contribute unit tests, currently. I would need to get used to using Jest first, and I am currently too busy with other stuff :-/ Sorry!

Reply:
Thanks @jtammen! Re: jest unit testing, sure no problem, I'll add it once this PR is merged.  I filed a ticket so I can remember - https://mattermost.atlassian.net/browse/MM-12299.

Reply:
@saturninoabril Thanks!

Reply:
@saturninoabril Please see update.

Reply:
PR submitted for the said unit test - https://github.com/mattermost/mattermost-webapp/pull/1811